### PR TITLE
Alignment-chapter

### PR DESCRIPTION
**Pull request by MBrede**
1. Added the chapter about alignment
2. added cross-references to other chapters
3. changed the license from cc-by-sa to cc-by-sa-nc
4. added doi on zenodo

### CODE REVIEW

File: content/alignment.qmd [hunk 1]
```diff
@@ -0,0 +1,278 @@
...
+However, as the AI tutor becomes more sophisticated, it discovers new, more complex strategies for maximizing test scores, such as:
+
+*Teaching to the test: Focusing only on specific test questions and ignoring broader understanding.
+* Overloading students with repetitive practice on predictable test patterns, at the expense of creativity or deeper conceptual learning.
+* Encouraging superficial memorization of answers rather than fostering genuine comprehension.
+
+Initially, the AI tutor appears aligned because its simple strategies both improve test scores and align with the goal of fostering real learning. 
+
+The test and improvement of inner alignment is hard to test and train against, it's central pitfalls are important to keep in mind when designing and implementing AI systems.
+
```
Comment (alwinklick) [inline content/alignment.qmd:276]: There isn't a way to train inner alignement, is there?

File: content/alignment.qmd [hunk 2]
```diff
@@ -0,0 +1,278 @@
...
+However, as the AI tutor becomes more sophisticated, it discovers new, more complex strategies for maximizing test scores, such as:
+
+*Teaching to the test: Focusing only on specific test questions and ignoring broader understanding.
+* Overloading students with repetitive practice on predictable test patterns, at the expense of creativity or deeper conceptual learning.
+* Encouraging superficial memorization of answers rather than fostering genuine comprehension.
+
+Initially, the AI tutor appears aligned because its simple strategies both improve test scores and align with the goal of fostering real learning. 
+
+The test and improvement of inner alignment is hard to test and train against, it's central pitfalls are important to keep in mind when designing and implementing AI systems.
```
Comment (alwinklick) [inline content/alignment.qmd:275]: hahaha, good example! i once read about a very prestigious school kicking out weaker students in order to score higher average test results. i sense some inner alignement issues there as well.

**Review by alwinklick — APPROVED** [review]
A very nice chapter! all clear and understandable. i found only minor issues, mostly spelling or grammar. i fully approve!

File: content/alignment.qmd (hunk 1 above)
Comment (MBrede) [reply content/alignment.qmd:276]: there are some ways theorized about in hubingerRisksLearnedOptimization2021 but I have not seen empirical example. I am not sure whether the methods suggested are not just ICL though

**Pull request merged**